insights/parsers/systemd/config.py

```diff
@@ -134,10 +136,20 @@ def parse_systemd_ini(content):
     for section in Config.sections():
         dict_section = {}
         for option in Config.options(section):
-            value = Config.get(section, option).splitlines()
-            value = map(lambda s: s.rstrip("\\").strip(), value)
-            value = filter(lambda s: bool(s), value)
-            dict_section[option] = "\n".join(value)
+            value = Config.get(section, option).splitlines()[::-1]
+            value_middle = []
+            while value:
+                value_pre = []
+                a = value.pop()
+                value_pre.extend([a])
+                while a.endswith("\\"):
+                    a = value.pop()
+                    value_pre.extend([a])
+                value_pre = map(lambda s: s.rstrip("\\").strip(), value_pre)
+                value_pre = filter(lambda s: bool(s), value_pre)
+                value_pre = [' '.join(value_pre)]
+                value_middle.extend(value_pre)
```

Comment:
@wushiqinlou you might be able to simplify this by using `insights.parsers.unsplit_lines`.